### PR TITLE
Clever native KV support

### PR DESCRIPTION
**Pull request by davlgd**
⚠️ This is an experimental project to show how easy it is to include native MateriaKV DB support to an application through the Redis API compatiblity layer ⚠️ 

Documentation is available [here](https://github.com/CleverCloud/clever-tools/blob/davlgd-embedded-kv-client/docs/kv.md).

### CODE REVIEW

File: docs/kv.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+clever kv ADDON_NAME_OR_ID TTL myKey                  # It will respond (integer) the remaining time to live of the key in seconds
+```
+
+>[!Tip]
+>You can get a list of all supported commands with `clever kv ADDON_NAME_OR_ID COMMANDS`
+
+You can pass the result of JSON stringified values to tools like `jq` to query them, for example:
+
+```bash
+clever kv ADDON_NAME_OR_ID SET myJsonFormatedKey {"key": "value"}
```
Comment (pdesoyres-cc) [inline docs/kv.md:39]: fix: Maybe safer with single quote around the json:

```suggestion
clever kv ADDON_NAME_OR_ID SET myJsonFormatedKey '{"key": "value"}'
```

**Review by pdesoyres-cc — DISMISSED** [review]
Nice one. I have some minor comments. I'm ok with this simple implementation.

We clearly have work to do around addon Ids resolution accros all commands related to addons.

File: src/commands/kv.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+/**
+ * Send a command to a compatible KV database
+ * @param {string} url
+ * @param {Array<string>} command
+ * @returns {Promise<string>} the command result
+ */
+async function sendCommand (url, command) {
+  Logger.debug(`Sending command '${command.join(' ')}' to ${url}`);
+  const client = new Redis(url, { maxRetriesPerRequest: MAX_RETRIES_PER_REQUEST });
```
[the remark this answers is not in the document]
Comment (davlgd) [reply src/commands/kv.js:77]: Do you want me to try/catch this or is there any ioredis specific way to get/throw errors ?

[the remark this answers is not in the document]
Comment (pdesoyres-cc) [reply src/commands/kv.js:77]: After thinking about it, we can let go every Error thrown by the io redis client.

**Review by hsablonniere — APPROVED** [review]
LGTM :+1:

**Pull request merged**